deis-cleanup.rb

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env ruby
+
+require "open3"
+require "csv"
+
+KEEP_LAST_VERSIONS = ENV["KEEP_LAST_VERSIONS"].to_i || 2
+DRY_RUN = ENV["DRY_RUN"] || false
```

Comment:
Line from README.md was removed because it may be missleading / hard to understand. And yes, both original and your solution will fail on 'false', I'm looking for something universal, so what about `YAML.load`.